docs/jegyzet/ado_net/ADO_NET.md

```diff
@@ -0,0 +1,275 @@
+# ADO&#46;NET #
+- [ADO&#46;NET](#adonet)
+  - [Mi is az az **ADO&#46;NET**?](#mi-is-az-az-adonet)
+  - [Connection](#connection)
+  - [Command](#command)
+    - [Előkészítés](#előkészítés)
+    - [Végrehajtás](#végrehajtás)
+      - [Tranzakciók](#tranzakciók)
+    - [NULL](#null)
+  - [Az eredmény](#az-eredmény)
+    - [DataReader](#datareader)
+    - [DataSet](#dataset)
+  - [Veszélyforrások](#veszélyforrások)
+    - [Connection string](#connection-string)
+    - [SQL injection](#sql-injection)
+    - [Connection leak](#connection-leak)
+
+## Mi is az az **ADO&#46;NET**? ##
+Adatvezérelt alkalmazásokban fontos, hogy az adatelérési réteg *kényelmes, egyszerű metodikákat* biztosítson az adatbázissal való kommunikációra, összetett lekérdezések könnyebb megfogalmazására, mindezt magától az *adatbázistól függetlenül*.
+
+A Microsoft által fejlesztett **ADO** (*ActiveX Data Object*) egy **adatelérési osztálykönyvtár**, mely pont ezeket az igényeket hivatott kielégíteni. A **.NET Framework** részeként gazdag eszköztárat szolgáltat adatvezérelt alkalmazások készítéséhez, egyszerű hozzáférést biztosítva az alkalmazásból relációs adatbázisokhoz, *függetlenül az adatbázis konkrét példányától és típusától*.

```

Comment:
Az ADO nem azonos az ADO.NET-tel https://www.geeksforgeeks.org/difference-between-ado-and-ado-net/

Reply:
done